tests/client/test_search_repo_conversions.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from pubtools.pulplib import Criteria, Matcher, Repository
+
+from pubtools.pulplib._impl.client.search import filters_for_criteria
+
+
+def test_eng_product_in():
```

Comment:
minor: 'test_eng_product_id'

Reply:
Actually I did mean "in" because the test is doing a search with "in" operator, though I see how it's ambiguous.

Reply:
I see what you mean. It's called eng_product_id, so looked like a typo.